token_server/README.md

```diff
@@ -24,19 +24,81 @@ you a command to run, probably `gcloud auth login`. Run it, logging in as
+   additional work is to create a service account that is whitelisted for
+   EarthEngine access (currently `earthengine-token-provider`) and give it the
+   "Service Account Token Creator" role in the
+   [IAM page](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis).
+
+*  If you are serving tokens from Google App Engine, and not from Amazon, then
+   you should also give the default AppEngine service account (currently
+   `mapping-crisis@appspot.gserviceaccount.com`) the "Service Account Token
+   Creator" role, so that it can create tokens for the service account you
+   created above.
+
+## Run server on Amazon Elastic Beanstalk
+
+*  Deploy using
+   [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
+   Since the project already exists on Amazon Elastic Beanstalk, you just need
+   to run `eb deploy` after running `eb init` in order to log in.
+
+*  The server URL will be entered here once we have GiveDirectly's account.
+
+*  See the [console](https://console.aws.amazon.com/) for more information. We
+   are using Elastic Beanstalk.
+
+### Set up new Amazon Elastic Beanstalk instance
+
+*  You should not normally need to do this if you already have a running server.
+*  See the [Google Cloud instructions](#set-up-new-google-cloud-instance) if
+   that is also changing.
+*  Delete the [.elasticbeanstalk](./.elasticbeanstalk) subdirectory of this
+   directory, and follow the  instructions
+   [here]((https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/)
+   to set up from scratch.
+*  Delete the `.gitignore` file that setup creates, and add
+   `.elasticbeanstalk/*` to git.
+*  Add Google service account credentials to
+   [Amazon Secrets Manager](https://aws.amazon.com/secrets-manager/getting-started/).
+   Remember that you need to be logged into Google as
+   `gd-earthengine-user@givedirectly.org`.
+     -  The Google service account is currently
+        `earthengine-token-provider@mapping-crisis.iam.gserviceaccount.com`
+        (specified in [ee_token_creator.js](./ee_token_creator.js)).
+     -  Download private key JSON for the Google service account via the
+        [Credentials page](https://console.developers.google.com/apis/credentials?project=mapping-crisis).
+     -  Upload to the [Amazon Secrets Manager](https://aws.amazon.com/secrets-manager/getting-started/)
+        using the provided instructions.
+     -  Take the "Secret ARN"'" (starting with `arn:aws:secretsmanager`) and put
+        it into the `SECRET_ID` field in
+        [aws_get_credentials.js](./aws_get_credentials.js).
+     - Make sure the region you created the secret in matches the region
+       specified in `AWS_REGION` in
+       [aws_get_credentials.js](./aws_get_credentials.js)! 
+   To make the secret readable by the server, you will have to have the server
```

Comment:
This paragraph is coming up as part of the previous bullet (at least according to the in review md preview page)

Reply:
Fixed, let me know if flow looks ok to you.